docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-mesh-deploy.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: mesheryctl system mesh deploy
+permalink: reference/mesheryctl/mesh/deploy
+type: reference
+display-title: "false"
+language: en
+# image: /assets/img/platforms/brew.png
+---
+
+<!-- Copy this template to create individual doc pages for each mesheryctl commands -->
+
+<!-- Name of the command -->
+# mesheryctl mesh deploy
+
+<!-- Description of the command. Preferably a paragraph -->
+## Description
+
+{% for command_hash in site.data.mesheryctlcommands.meshes.deploy.command %}{% assign command = command_hash[1] %}
+{{ command.description }}
+{% endfor %}
+
+<!-- Basic usage of the command -->
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  mesheryctl mesh deploy
```

Comment:
It's basic usage won't work and will return a message saying that parameters are missing.￼

Reply:
// @alphaX86, @sayantan1413 .

Reply:
Seems we need to try the command and get back to you....

Reply:
Gents, ping when this has been addressed.

Reply:
Copy 👍

Reply:
we need to update the docs first! what do u think? // @alphaX86

Reply:
We'll let you know soon by today @afzal442 I'm a bit free now from exams so I'll tell you what to do next

Reply:
@afzal442 I just verified the command, it needs a Token for k8s to be run, else it reports as no token found. So we need to extend that a bit

Reply:
Say, I ran the command in blank it shows the below O/P...
![image](https://user-images.githubusercontent.com/59508546/130085859-e021b567-4821-4417-a3e7-9bbffd2c2d66.png)

So, we need to add flags for proper usage

Your opinions? 
// @leecalcote  @sayantan1413

Reply:
`mesheryctl mesh deploy --adapter adapter-name:port -t ~/Downloads/auth.json "service_mesh_name"` and the `service_mesh_name must be SMP protos compliant`. This is the command with all the parameters here.

Reply:
so could you update mesheryctl command tracker docs for that? thanks //@alphaX86 @sayantan1413

Reply:
The spreadsheet? Hmm... it can be done after changing the docs. Coz it's the main source of truth

Reply:
gr8! @alphaX86 thanks.